app/Console/Commands/RebuildCardsTable.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Http;
+use JsonMachine\Items;
+use App\Models\Card;
+
+class RebuildCardsTable extends Command
+{
+    protected $signature = 'scryfall:rebuild-cards';
+    protected $description = 'Rebuild the cards table daily using Scryfall bulk data';
```

Comment:
I'm not seeing in this commit where it sets the timing of the runs but I'd say we should only run monthly or bi monthly (effectively when there's a new set)

Reply:
My understanding is that this is meant to be run manually to seed your local DB just to prove that it works. Although, I can see it being set up as a cron.

@jordyarms Is the description accurate or is it LLM generated?

Reply:
I dropped the cron job and script from the pull request, I had it written but scaled back for the phase as we're in early test and learn. 

Scryfall specifies their change cadence as a 12 hr rotation described on https://scryfall.com/docs/api/bulk-data
The initial discovery work posed questions on the design of the service such as if the table should be reset each time – those details can be decided later, it's out of scope for right now.

Reply:
I agree with this. Honestly I think even the command is a bit much for this PR. But I also think it's useful. I'm good with not having the cron in this PR.

Reply:
I just don't want us to get got by making too many calls accidentally